docs/technical_reference/unit_models/uv_aop.rst

```diff
@@ -0,0 +1,96 @@
+
+1) UV light and hydrogen peroxide (:math:`\text{H}_2\text{O}_2`)
+
+2) UV light and ozone (:math:`\text{O}_3`)
+
+3) UV light, hydrogen peroxide (:math:`\text{H}_2\text{O}_2`) and ozone (:math:`\text{O}_3`)
+
+UV AOPs can be  modeled at several different levels, depending on the known kinetic pathways and the modeling objectives. In this work, a basic level kinetic model is presented with an assumption on pseudo-steady state approximation for the kinetic description of free radical species. A pseudo-first order rate constant is utilized to represent the overall degradation rate of contaminants. The users need to provide either UV dose and disinfection rate, or rate constant and exposure time that are can be acquired during UV validation tests. These measurements are then used to simulate the removal of contaminants during the UV AOP process.
+
+This model also accounts for the energy demand for the disinfection process. The users need to provide Electrical Efficiency per Log Order Reduction (EE/O) to evaluate energy costs.
+
+Ports
+-----
+
+The model provides two ports (Pyomo notation in parentheses):
+
+* Inlet port (inlet)
+* Outlet port (outlet)
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "Time", ":math:`t`", "[0]"
+   "Phases", ":math:`p`", "['Liq']"
+   "Components", ":math:`j`", "['H2O', 'NDMA', ...]"
+
+Degrees of Freedom and Variables
+--------------------------------
+The UV system includes the state variables from the associated property package, and an outline of these variables are listed below:
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
+
+   "Inlet temperature", ":math:`T`", "temperature", "[t]", ":math:`\text{K}`"
+   "Inlet pressure", ":math:`P_{in}`", "pressure", "[t]", ":math:`\text{Pa}`"
+   "Outlet pressure", ":math:`P_{out}`", "pressure", "[t]", ":math:`\text{Pa}`"
+   "Mass flowrate of components", ":math:`M_j`", "flow_mass_phase_comp", "[t, 'Liq', j]", ":math:`\text{kg/s}`"
+   "Inlet volumetric flowrate", ":math:`F_{in}`", "flow_vol", "[t]", ":math:`\text{m}^3\text{/s}`"
+
+**NOTE: Variables for 'temperature', 'pressure', 'flow_mass_phase_comp', and 'flow_vol' come from the associated property package as state variables and are accessed via {port_name}.{state_var_name}**
+
+Aside from the inlet feed state variables (i.e., temperature, pressure, component mass flowrates),
+the UV AOP model has at least an additional 5 degrees of freedom that
```

Comment:
This implies that the user must fix 5 variables, but the table lists more than 5. So if a user follows these instructions, wouldn't they overspecify the problem?

Also, above you mention "users need to provide either UV dose and disinfection rate, or rate constant and exposure time that are can be acquired during UV validation tests." While there can be different combinations of which DOF to fix, maybe we should provide a list of the typical variables to fix for a simulation.

Reply:
I've specified which variable should be fixed from the list in the note of new version.